### PR TITLE
Add a format for @InCollection.

### LINKED ISSUE #397: broken formatting
these entries are formatted incorrectly starting with ", " in the second line:
image

@bangerth You know the formatting schema best. Do you know what is happening here?

### PR DESCRIPTION
**Pull request by bangerth**
Should fix #397.

### CODE REVIEW

File: jabref-template/listrefs.incollection.layout [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        comment-end marker. 
+
+        Everything is closed with a period after the last entry.
+
+        -->
+
+       \begin{booktitle}In: \format[HTMLChars]{\booktitle}\end{booktitle}<!--
+       -->\begin{editor} (\format[HTMLChars]{\editor})\end{editor}<!--
+        -->\begin{pages}, pp. \format[FormatPagesForHTML]{\pages}\end{pages}<!--
+    -->\begin{publisher}, \format[HTMLChars]{\publisher}\end{publisher}<!--
```
Comment (marcfehling) [inline jabref-template/listrefs.incollection.layout:34]: I would also display the `note` field as in the other layouts.
```suggestion
        -->\begin{pages}, pp. \format[FormatPagesForHTML]{\pages}\end{pages}<!--
         -->\begin{note}  (\format[HTMLChars]{\note})\end{note}<!--
    -->\begin{publisher}, \format[HTMLChars]{\publisher}\end{publisher}<!--
```

**Review by marcfehling — APPROVED** [review]
Let's give it a try!

File: jabref-template/listrefs.incollection.layout (hunk 1 above)
Comment (marcfehling) [reply jabref-template/listrefs.incollection.layout:34]: We can also debate whether the `volume` and `series` flag make sense, for example for this article:
https://github.com/dealii/publication-list/blob/f8e1b2d0c80361e1df64684bef278a936d01edc9/publications-2016.bib#L741-L752

Essentially, `InCollection` should be treated like `InProceedings`. I wonder whether `jabref` supports the use of one layout for multiple types.

Comment (bangerth) [reply jabref-template/listrefs.incollection.layout:34]: No, it does not. I copied this one from `@InProceedings`. Let's not make things more complicated than necessary. This is a step forward, it doesn't have to be perfect.

**Review by marcfehling — APPROVED** [review]

**Pull request merged**